.github/settings.yml

```diff
@@ -100,41 +108,55 @@ labels:
 - name: component/ocm-website
   color: 25399e
   description: https://ocm.software/
-- name: component/vscode-extension
+- name: component/github-actions
   color: 25399e
-  description: MS Visual Studio Code OCM Extension
-
-- name: kind/bug
+  description: Changes on GitHub Actions or within `.github/` directory
+  oldname: github-actions
+## The different kinds of issues and pull requests - will be used for release notes grouping
+- name: kind/bugfix
   color: B60205
   description: Bug
-- name: kind/fix
-  color: c7def8
-  description: Bug fix
+  oldname: kind/bug
```

Comment:
we have a problem here because we have multiple oldnames ... what about `fix`

Reply:
hmmm...why should we label anything as fix or kind/fix? This would only be used at PRs right?

Reply:
Any kind of bugfix is usually used with one of `kind/bug` `kind/fix` or `fix` I believe. If we unify im fine but I want to make sure that we have it working correctly if there are multiple `oldname` candidates. no idea...
maybe we have to manually migrate

Reply:
unfortunately yes - we have to manually migrate - the oldname thing isn't really documented (it only occurs in some samples) and it's not working as expected (at least during my tests)